pkg/docker-engine/Dockerfile

```diff
@@ -188,7 +188,7 @@ ARG DOCKER_ENGINE_REF
 ARG NIGHTLY_BUILD
 WORKDIR /build
 ARG TARGETPLATFORM
-RUN xx-apt-get install -y gcc libc6-dev libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev libsecret-1-dev libsystemd-dev libudev-dev pkg-config
+RUN xx-apt-get install -y gcc libc6-dev libapparmor-dev libltdl-dev libseccomp-dev libsecret-1-dev libsystemd-dev libudev-dev pkg-config
```

Comment:
interesting actually that we had to manually install these; didn't we let the rpm/deb tools handle install dependencies?

Reply:
This stage is to build static binaries like https://github.com/docker/docker-ce-packaging/pull/835

Reply:
In the future if I can rely on upstream Dockerfiles I could just use the right remote target and avoid redefining this stage.

Reply:
Oh! Yes for the static binaries it makes sense